src/Jobs/BestPracticesUpdater.js

```diff
@@ -0,0 +1,31 @@
+const BaseJob = require('../Structure/BaseJob');
+const axios = require('axios');
+
+class BestPracticesUpdater extends BaseJob {
```

Comment:
This job should run during the application start. We should create a new `Initializers` directory. The migrations check can move to a file there (throw an error if migrations are needed, so it aborts the app boot) & running this job can also be a script in there.